lib/cqm/converter/hds_record.rb

```diff
@@ -108,6 +98,22 @@ def to_qdm(record)
       patient.family_name = record.last if record.last
       patient.bundle_id = record.bundle_id if record.bundle_id
 
+      # Convert extended_data.
```

Comment:
Bonnie team (@jbradl11 or whoever actually reviews this): the relevance of these `extended_data` attributes to Bonnie is something we should look at. If there are any that are useful to you as well as Cypress, we should pull them out into the core `Patient` object.